WalletWasabi/WabiSabi/Client/BobClient.cs

```diff
@@ -29,5 +29,18 @@ public async Task RegisterOutputAsync(Money amount, Script scriptPubKey, IEnumer
 				vsizeCredential,
 				cancellationToken).ConfigureAwait(false);
 		}
+
+		public async Task<ArenaResponse> ReissuanceAsync(Money amount1, int scriptPubKeyVsize1, Money amount2, int scriptPubKeyVsize2, IEnumerable<Credential> amountCredential, IEnumerable<Credential> vsizeCredential, CancellationToken cancellationToken)
```

Comment:
same as above, I wouldn't mind `ReissuanceAsync(IEnumerable<Money> amountsToRequest, IEnumerable<int> vsizesToRequest, IEnumerable<Credential> amountCredentialsToPresent, IEnumerable<Credential> vsizeCredentialsToPresent, CancellationToken ct)` but I would prefer `IEnumerable<ulong>` only at this level too